docs/index.md

```diff
@@ -1,71 +1,126 @@
-Open Data Services Sphinx Base
-==============================
+Standardising Interest Declarations
+===================================
 
+## Contents
+
+```eval_rst
+.. toctree::
+   :maxdepth: 3
+
+   index
+
+```
+
+## Background
+
+In early 2020, Open Data Services Co-operative and OpenDemocracy collaborated on a project to create a searchable, public database of politicians' and elected members' interests. This resulted in [declared.info](http://declared.info/).
+
+Declarations of interest were obtained by scraping diverse websites plus examining registers of interest provided in response to Freedom of Information requests. The diversity of formats, detail and quality represented by these declarations made work on [declared.info](http://declared.info/) unnecessarily challenging, and the site itself not as useful as it might be. Moreover, the format of the data means that it cannot be easily merged or compared with other datasets.
+
+Public declarations of potential conflicts of interest are a required component of the checks and balances in a democratic society. When these declarations only have value in isolation and cannot easily be compared with others, or linked to other types of information, they have limited value. Especially so at a time when private sector and non-democratic actors face no such limitations.
+
+If democratic institutions and bodies adopted an open data standard for publishing their registers of interests, tools such as [declared.info](http://declared.info/) could be built cheaply and have far-reaching scope. The public interest would be far better served.
+
+## Status of the data model
+
+We present here the beginnings of an open data standard: a draft data model to represent information declared by people about their interests. The structure of the data (the data model) is informed by:
+
+* the experience of dealing with unstructured, variously formatted information in this domain,
```

Comment:
I think that semicolons can be used at the end of sub-clauses after a colon, meaning that commas used to separate list items aren't ambiguous

Reply:
Done